### DISCUSSION TITLE
Edited Image Path!

### DISCUSSION THREAD

**Discussion by brittle123** [burhanrashid52/PhotoEditor#362] [Q&A]
Hello Sir!,... you're saving your edited image in PICTURES directory i think but i can't find in which file you're saving and giving the path for it. Please do let me know where and what do i've to change to save my edited image in custom folder name.
Thanks & Regards

**Comment by burhanrashid52** [comment]
You can check `EditImageActivity` or `FileSaveHelper` to saving image to specific path.
